### DISCUSSION TITLE
System mails via S/MIME and OpenPGP

### DISCUSSION THREAD

**Discussion by linux-lukas** [mastodon/mastodon#18355] [General]
Thought process: It would be possible for an S/MIME certificate or a public OpenPGP key (optionally with .sig file?) to be stored for the e-mail address stored in the profile. The Mastodon system would then send every email triggered by the instance to the email encrypted via S/MIME or OpenPGP.

**Comment by ClearlyClaire** [comment]
Seems like a duplicate of https://github.com/mastodon/mastodon/issues/4501
